src/tests/fableconfig.json

```diff
@@ -13,7 +13,7 @@
             "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
             "module": "commonjs",
-            "coreLib": "../../build/fable-core/umd",
+            "coreLib": "../../build/fable-core",
```

Comment:
Ah, ok! I didn't remember that for the `FableCompilerNetcore` target, fable-core TS files were compiled directly in commonjs format. Thanks for the explanation! :+1: